core/graph/src/main/java/uk/gov/gchq/gaffer/graph/Graph.java

```diff
@@ -358,6 +380,26 @@ public Builder addSchema(final InputStream schemaStream) {
             }
         }
 
+        public Builder addSchema(final URL schemaURL) {
```

Comment:
I don't think we need these builder methods. Users can just use the StreamUtil class to convert a URL into an InputStream and then use the addSchema(final InputStream schema) method.